package/MDAnalysis/lib/distances.py

```diff
@@ -1440,17 +1493,42 @@ def calc_bonds(coords1: Union[np.ndarray, 'AtomGroup'],
         if box is not None:
             boxtype, box = check_box(box)
             if boxtype == 'ortho':
-                _run("calc_bond_distance_ortho",
-                     args=(coords1, coords2, box, bondlengths),
-                     backend=backend)
+                if backend == 'distopia':
+                    # need explicit branch on backend to prepare input types correctly
+                    # must assign to result to get correct reference on memview
+                    bondlengths = _run("calc_bonds_ortho_float",
+                         args=(coords1, coords2, box[:3]),
+                         kwargs={'results': bondlengths.astype(np.float32)},
+                         backend=backend)
+                    # upcast is currently required, change for 3.0
+                    bondlengths = bondlengths.astype(np.float64)
+                else:
+                    _run("calc_bond_distance_ortho",
+                         args=(coords1, coords2, box, bondlengths),
+                         backend=backend)
             else:
+                if backend == 'distopia':
+                    # distopia does not support triclinic boxes
+                    backend_ = 'serial'
```

Comment:
Is this documented somewhere? (I'm not seeing anything). We probably don't want to raise a warning every time it happens, but folks should be aware.

Reply:
I added a `warning` block.